crates/bevy_transform/README.md

```diff
@@ -1,3 +1,5 @@
 # Bevy Transform
 
-This crate is largely a 1:1 port from [legion_transform](https://github.com/AThilenius/legion_transform) (ecs: legion, math: nalgebra) to bevy (ecs: bevy_ecs, math: glam)
+This crate was largely a 1:1 port from [legion_transform](https://github.com/AThilenius/legion_transform) (ecs: legion, math: nalgebra) to bevy (ecs: bevy_ecs, math: glam)
```

Comment:
I think this line is just fully obsolete at this point; we've changed the data structures around since then.